services/PDFService/main.py

```diff
@@ -30,7 +30,7 @@
 
 # Configuration
 MODEL_API_URL = os.getenv(
-    "MODEL_API_URL", "https://nv-ingest-rest-endpoint.brevlab.com/v1"
```

Comment:
this appears to only be available if you also run the calling services on a similar brevlab network or something